core/common/src/test/java/alluxio/ConfigurationTest.java

```diff
@@ -735,4 +736,26 @@ public void toRawMap() throws Exception {
     assertTrue(confRegex.matcher(rawMap.get("alluxio.locality.script")).find());
     assertTrue(confRegex.matcher(rawMap.get("alluxio.logs.dir")).find());
   }
+
+  @Test
+  public void getDisplayValue() {
+    PropertyKey testKey = PropertyKey.S3A_SECRET_KEY;
+    String testValue = "12345";
+    assertEquals(PropertyKey.DisplayType.CREDENTIALS, testKey.getDisplayType());
+    Configuration.set(testKey, testValue);
+
+    assertNotEquals(testValue, Configuration.getDisplayValue(testKey));
+    assertNotEquals(testValue, Configuration.toMap(true).get(testKey.getName()));
+  }
+
+  @Test
+  public void getNestedDisplayValue() {
+    PropertyKey nestedProperty =
+        PropertyKey.fromString("alluxio.master.journal.ufs.option.aws.secretKey");
+    String testValue = "12345";
+    Configuration.set(nestedProperty, testValue);
+
+    assertNotEquals(testValue, Configuration.getDisplayValue(nestedProperty));
+    assertNotEquals(testValue, Configuration.toMap(true).get(nestedProperty.getName()));
+  }
```

Comment:
some additional cases to test: 
- nested non-credentials are displayed like normal
- non-nested non-credentials are displayed like normal
- display values for credentials are the same, even if the credentials are different
- `getRawMap(true)` hides credentials

Reply:
I added some more tests.